scripts/kube-upgrade.sh

```diff
@@ -32,9 +33,14 @@ run_upgrade() {
     # Once a node creates a configmap, other nodes will remain at this step until the first node deletes the configmap when upgrade completes.
     if [ "$NODE_ROLE" != "worker" ]
     then
-      until kubectl --kubeconfig /etc/kubernetes/admin.conf create configmap upgrade-lock > /dev/null
+      until kubectl --kubeconfig /etc/kubernetes/admin.conf create configmap upgrade-lock --from-literal=node="${CURRENT_NODE_NAME}" > /dev/null
```

Comment:
@kpiyush17 better to create cm in `spectro-system` or `kube-system` or  namespace